libsession/build.gradle

```diff
@@ -11,6 +12,8 @@ android {
 
     defaultConfig {
         minSdkVersion androidMinimumSdkVersion
+        // Build constant for which version should minimally support new groups
+        buildConfigField("String", "MINIMUM_GROUP_VERSION", '"1.19.0"')
```

Comment:
Is this used? If so I don't think this version number is right

Reply:
No I think the text has been updated to just say "latest version". Deleted